constants/quizPrice.ts

```diff
@@ -0,0 +1,42 @@
+export const priceToTemplates = [
```

Comment:
That looks cool =))
But there is a better way)

```suggestion
export const priceToTemplates = Array.from({ length: 38 }, (_, i) => {
  const value = 200000 - i * 5000;
  return { value, label: `${value.toLocaleString()} $` };
})
```